cts/cts-cli.in

```diff
@@ -2289,6 +2289,38 @@ function test_tools() {
     desc="Switch to nonexistent shadow instance (force) (XML)"
     cmd="crm_shadow --switch $shadow --batch --force --output-as=xml"
     test_assert_validate $CRM_EX_NOSUCH 0
+    
+    CIB_file_invalid_1="$test_home/cli/crm_verify_invalid_bz.xml"
+    CIB_file_invalid_2="$test_home/cli/crm_verify_invalid_no_stonith.xml"
+    CIB_file_invalid_3="$test_home/cli/crm_verify_invalid_xml_error.xml"
+
+    desc="Verbosely verify a file-specified invalid configuration, ouputting as xml"
+    cmd="crm_verify --xml-file $CIB_file_invalid_1 --output-as=xml --verbose"
```

Comment:
This isn't a big deal in this case, but just for good shell coding practices, quote "$CIB_file_invalid_1" (and similar for the other crm_verify commands). The idea is that if $test_home has a space in it, it will work quoted but will appear to be multiple arguments otherwise.